packages/component-library-angular/src/button-link/ButtonLink.spec.ts

```diff
@@ -0,0 +1,190 @@
+
+      expect(buttonLink).toHaveClass('utrecht-button-link--secondary-action');
+    });
+
+    it('renders a design system BEM `utrecht-button-link--subtle-action` class', async () => {
+      const { container } = await render(
+        '<a href="https://example.com" [appearance]="appearance" utrecht-button-link></a>',
+        {
+          declarations: [UtrechtButtonLinkAttr],
+          componentProperties: {
+            appearance: 'subtle-button',
+          },
+        },
+      );
+      const buttonLink = container.querySelector('.utrecht-button-link');
+
+      expect(buttonLink).toHaveClass('utrecht-button-link--subtle-action');
+    });
+    it('can not renders a design system BEM `utrecht-button-link--primary-action` class by default', async () => {
+      const { container } = await render('<a href="https://example.com"  utrecht-button-link></a>', {
+        declarations: [UtrechtButtonLinkAttr],
+      });
+      const buttonLink = container.querySelector('.utrecht-button-link');
+
+      expect(buttonLink).not.toHaveClass('utrecht-button-link--primary-action');
+    });
+    it('can not renders a design system BEM `utrecht-button-link--secondary-action` class be default', async () => {
+      const { container } = await render('<a href="https://example.com" utrecht-button-link></a>', {
+        declarations: [UtrechtButtonLinkAttr],
+      });
+      const buttonLink = container.querySelector('.utrecht-button-link');
+
+      expect(buttonLink).not.toHaveClass('utrecht-button-link--secondary-action');
+    });
+
+    it('can not renders a design system BEM `utrecht-button-link--subtle-action` class by default', async () => {
+      const { container } = await render('<a href="https://example.com" utrecht-button-link></a>', {
+        declarations: [UtrechtButtonLinkAttr],
+      });
+      const buttonLink = container.querySelector('.utrecht-button-link');
+
+      expect(buttonLink).not.toHaveClass('utrecht-button-link--subtle-action');
+    });
+  });
+
+  describe('Button Link `external`', () => {
+    it('prevents sharing referer information', async () => {
+      await render('<a utrecht-button-link [external]="external" href="https://example.com">Link</a>', {
+        declarations: [UtrechtButtonLinkAttr],
+        componentProperties: {
+          external: true,
+        },
+      });
+
+      const buttonLink = screen.getByRole('link', { name: 'Link' });
+
+      expect(buttonLink).toHaveAttribute('rel', 'external noopener noreferer');
+    });
+
+    it('prevents access to window.opener', async () => {
+      await render('<a utrecht-button-link [external]="external" href="https://example.com">Link</a>', {
+        declarations: [UtrechtButtonLinkAttr],
+        componentProperties: {
+          external: true,
+        },
+      });
+      const buttonLink = screen.getByRole('link', { name: 'Link' });
+      expect(buttonLink.getAttribute('rel')).toContain('noopener');
```

Comment:
`toContain('noopener')` would also work if the attribute was `rel="brokennoopener"`, so I would use a method that actually works with values in a whitespace separated list.